{{cookiecutter.project_slug}}/.readthedocs.yml

```diff
@@ -1,12 +1,15 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
 version: 2
 
+# Build documentation in the docs/ directory with Sphinx
 sphinx:
   configuration: docs/conf.py
 
-build:
-  image: testing
-
+# Version of Python and requirements required to build the docs
 python:
-  version: 3.10
+  version: "3.8"
```

Comment:
We should run with Python 3.10, not 3.8, plus this option is now [deprecated](https://docs.readthedocs.io/en/stable/config-file/v2.html#python-version)
```suggestion
```